jenkins/prepare-ps-build-docker.yml

```
@@ -52,16 +52,12 @@
                 stage('Build') {
                     steps {
                         parallel(
-                            "i386/centos:6":  { build('i386/centos:6') },
-                            "centos:6":       { build('centos:6') },
-                            "centos:7":       { build('centos:7') },
-                            "centos:8":       { build('centos:8') },
-                            "ubuntu:artful":  { build('ubuntu:artful') },
-                            "ubuntu:xenial":  { build('ubuntu:xenial') },
+                            "centos7":  { build('centos:7') },
+                            "centos8":  { build('centos:8') },
```

looks like missed colon
```suggestion
                            "centos:7":  { build('centos:7') },
                            "centos:8":  { build('centos:8') },
```

---

@delgod done